.github/CODEOWNERS

```diff
@@ -111,33 +111,45 @@
 ########## Plugins ##########
 
 /plugins/btauto/       @coq/btauto-maintainers
+/theories/btauto/      @coq/btauto-maintainers
 
 /plugins/cc/           @coq/cc-maintainers
+/theories/cc/          @coq/cc-maintainers
 
 /plugins/derive/       @coq/derive-maintainers
+/theories/derive/      @coq/derive-maintainers
 
 /plugins/extraction/   @coq/extraction-maintainers
+/theories/extraction/  @coq/extraction-maintainers
 
 /plugins/firstorder/   @coq/firstorder-maintainers
+/theories/firstorder/  @coq/firstorder-maintainers
 
 /plugins/funind/       @coq/funind-maintainers
+/theories/funind/      @coq/funind-maintainers
 
 /plugins/ltac/         @coq/ltac-maintainers
+/theories/ltac/        @coq/ltac-maintainers
 
 /plugins/micromega/    @coq/micromega-maintainers
+/theories/micromega/   @coq/micromega-maintainers
 /test-suite/micromega/ @coq/micromega-maintainers
 
 /plugins/nsatz/        @coq/nsatz-maintainers
-
 /plugins/setoid_ring/  @coq/ring-maintainers
```

Comment:
Same question for `theories/setoid_ring/`.

Reply:
Fixed, thanks.